content/post/2023-02-03-friday-hacks-234.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Friday Hacks #234, February 03: Indie Developer and Hack&Roll 2023 Projects"
+date: 2023-01-29 13:25:15.5144
+author: Toh Li Heng
+url: /2023/02/friday-hacks-234
+summary: From idea to revenue and some cool hacks from Hack&Roll 2023.
+sponsors:
+    - hangar
+---
+
+**Date/Time:** Friday, January 27 at 7:00pm SGT<br />
+**Venue:** [**Level 2 Auditorium** @ i3 Building](https://goo.gl/maps/aUMwWptKFsajR93b9)<br />
+**Register for Food:** [NUSync Food Sign-up Link](https://hckr.cc/fh2223s2-w4)<br />
+
+> **Food 🍕 will be served!**
+
+_If you are unable to come join us physically, you are welcome to join us [online](https://hckr.cc/fh2223s2-w4-zoom)!_
+
+<img src="/img/2023/fh/234.jpg" alt="Friday Hacks #234 Poster" /><br />
+
+## 1) Idea to Revenue: How to make your first internet dollar
+
+Dexter will share his journey building Homerow, a productivity macOS application that has generated $5,000 in revenue since August 2022. He hopes to encourage Computing students to start building side projects to learn and  have fun 👀.
+
+### Speaker Profile
+
+Dexter is an incoming NUS CS freshman. He is currently working an 8-5 job. In his free time, he likes to build apps and occasionally sells them.
+
+## 2) Hack&Roll 2023: Winners
```

Comment:
The title doesn't match the pubs image which says "How to build an award winning project?"

I personally like "How to build an award winning project?" more than "Hack&Roll 2023: Winners"

You can mention Hack&Roll 2023 in the description: "IOT systems, these projects from Hack&Roll 2023..."